musicAlbum/musicalbum.rb

```diff
@@ -0,0 +1,12 @@
+require_relative '../item'
+
+class MusicAlbum
+  def initialize(_on_spotify)
+    super()
+    @on_spotify = false
+  end
+
+  def can_be_archived?()
+    @archived = true if Item.can_be_archived? && @on_spotify
+  end
+end
```

Comment:
- [ ] I think to call a method from the parent class you could just call `super`. Because when you call `super` it is going to call the same method from the parent class. So please try to use super instead of `Item.can_be_archived`.

Reply:
Thank you for pointing this one out. I have resolved it.